src/platforms/javascript/guides/nextjs/manual-setup.mdx

```diff
@@ -391,3 +393,23 @@ const moduleExports = {
 ```
 
 Excluded routes can be specified either as regexes or strings. When using a string, make sure that it matches the route exactly, and has a leading slash but no trailing one.
+
+## Configure Tunneling to avoid Ad-Blockers
+
+_(New in version 7.25.0)_
+
+You might notice that Sentry events are sometimes blocked by Ad-Blockers.
+Ad-blockers can be circumvented by using <PlatformLink to="/troubleshooting/#dealing-with-ad-blockers">tunneling</PlatformLink>.
```

Comment:
l: Should we move this line to the bottom of this section? Something like

If you don't want to use the automatic option, you can also circumvent ad blockers by using...